### DISCUSSION TITLE
Sensor Descriptions

### DISCUSSION THREAD

**Discussion by jouster1974** [davidrapan/ha-solarman#20] [General]
HI There...

as per the last post from me regarding your great new integration and possible tweking of sensor descriptions etc, Ive posted here as you suggested.

If you need any specific information from me, please do let me know....

**Comment by davidrapan** [comment]
At first you can then tell me the following:
- Inverter profile (definition file which you are using)
- List of sensors which are displayed for you as something like: "[0x?]" (together with some observed (possible) values which they had if their purpose is somewhat opaque)
